spec/models/equipment_item_spec.rb

```diff
@@ -1,160 +1,269 @@
-    # unnecessary at the moment
-    it 'ensures unique serials scoped to equipment model if it exists' do
-      em1 = FactoryGirl.create(:equipment_model)
-      em2 = FactoryGirl.create(:equipment_model)
-      FactoryGirl.create(:equipment_item, equipment_model: em1, serial: 'a')
-      em1_a2 = FactoryGirl.build(:equipment_item, equipment_model: em1,
-                                                  serial: 'a')
-      em1_nil1 = FactoryGirl.build(:equipment_item, equipment_model: em1,
-                                                    serial: nil)
-      em1_nil2 = FactoryGirl.build(:equipment_item, equipment_model: em1,
-                                                    serial: nil)
-      em1_blank1 = FactoryGirl.build(:equipment_item, equipment_model: em1,
-                                                      serial: '')
-      em1_blank2 = FactoryGirl.build(:equipment_item, equipment_model: em1,
-                                                      serial: '')
-      em2_a = FactoryGirl.build(:equipment_item, equipment_model: em2,
-                                                 serial: 'a')
-
-      expect(em1_a2.valid?).to be_falsey
-      expect(em1_nil1.save!).to be_truthy
-      expect(em1_nil2.valid?).to be_truthy
-      expect(em1_blank1.save!).to be_truthy
-      expect(em1_blank2.valid?).to be_truthy
-      expect(em2_a.valid?).to be_truthy
+  describe 'serial' do
+    it 'can be blank' do
+      item = FactoryGirl.build_stubbed(:equipment_item, serial: '')
+      expect(item.valid?).to be_truthy
+    end
+    it 'can be nil' do
+      item = FactoryGirl.build_stubbed(:equipment_item, serial: nil)
+      expect(item.valid?).to be_truthy
     end
+    it 'cannot be the same as another item of the same model' do
+      model = FactoryGirl.create(:equipment_model)
+      FactoryGirl.create(:equipment_item, equipment_model: model, serial: 'a')
+      item = FactoryGirl.build(:equipment_item, equipment_model: model,
+                                                serial: 'a')
+      expect(item.valid?).to be_falsey
+    end
+    it 'can be the same as another item of a different model' do
+      FactoryGirl.create(:equipment_item, serial: 'a')
+      item = FactoryGirl.build(:equipment_item, serial: 'a')
+      expect(item.valid?).to be_truthy
+    end
+  end
 
+  it 'saves an empty string value as nil for deleted_at field' do
     # this test passes even without the nilify_blanks call in the model, maybe
     # delete the call?
-    it 'saves an empty string value as nil for deleted_at field' do
-      @item.deleted_at = '   '
-      @item.save
-      expect(@item.deleted_at).to eq(nil)
+    item = FactoryGirl.build(:equipment_item)
+    item.deleted_at = '   '
+    item.save
+    expect(item.deleted_at).to eq(nil)
+  end
+
+  describe '#active' do
+    it 'returns active equipment items' do
+      active = FactoryGirl.create(:equipment_item)
+      FactoryGirl.create(:deactivated)
```

Comment:
The factory isn't actually used anywhere else, so changing the name is totally feasible!